releases/release-1.32/release-notes/maps/pr-128190-map.yaml

```
@@ -0,0 +1,42 @@
+pr: 128190
+releasenote:
+  text: Added support to the kube-apiserver for an alpha feature enabling external
+    signing of service account tokens and fetching of public verifying keys, by enabling
+    the alpha `ExternalServiceAccountTokenSigner` feature gate and specifying `--service-account-signing-endpoint`.
```

Let's standardize the Alpha/Beta/Stable: https://github.com/kubernetes/sig-release/pull/2666#discussion_r1861000949 

```suggestion
    the Alpha `ExternalServiceAccountTokenSigner` feature gate and specifying `--service-account-signing-endpoint`.
```